src/routes/volume/index.js

```diff
@@ -541,6 +541,15 @@ class Volume extends React.Component {
           },
         })
       },
+      showUpdateReplicaDiskSoftAntiAffinityModal(record) {
+        dispatch({
+          type: 'volume/showUpdateReplicaSoftAntiAffinityModal',
```

Comment:
Is this missing the "Disk" in "ReplicaDiskSoftAntiAffinity" on purpose, because the type is more general?  Or just a typo?

Reply:
If I understand correctly, there is a general showUpdateReplicaSoftAntiAffinityModal, but we use the softAntiAffinityKey to control its behavior. See https://github.com/longhorn/longhorn-ui/pull/642/files/0fd9df46150e7fed23e7911b33c9c95e5f541e50#diff-9cc47b9984589ba3c8b23a332e9f0a0520a1382d3c24d13730894e28cf2fe311R535-R543 (the lines just above) for how it is handled for Replica Zone Soft Anti Affinity.